### PR TITLE
feat: move cosmwasm into stable and x/incentive into experiemntal

### PR DESCRIPTION
**Pull request by robert-zaremba**
We are finalizing the cosmwams release, so need to swap the experiemntal features.

### CODE REVIEW

**Review by toteki — APPROVED** [review]

**Comment by toteki** [comment]
make sure to test the upgrade again before canon-2, to make sure we're not missing a line

File: app/app.go [hunk 1]
```diff
@@ -628,7 +631,34 @@ func New(
...
+	var err error
+	wasmDir := filepath.Join(homePath, "wasm")
+	app.wasmCfg, err = wasm.ReadWasmConfig(appOpts)
+	if err != nil {
+		panic(fmt.Sprintf("error while reading wasm config: %s", err))
+	}
+
+	// The last arguments can contain custom message handlers, and custom query handlers,
+	// if we want to allow any custom callbacks
+	availableCapabilities := "iterator,staking,stargate,cosmwasm_1_1"
```
Comment (gsk967) [inline app/app.go:643]:
```suggestion
	availableCapabilities := "iterator,staking,stargate,cosmwasm_1_1,umee"
```
can you add `umee` also

File: app/app.go [hunk 2]
```diff
@@ -477,7 +476,11 @@ func New(
 		app.LeverageKeeper,
 	)
 	app.LeverageKeeper.SetTokenHooks(app.OracleKeeper.Hooks())
-	app.LeverageKeeper.SetBondHooks(app.IncentiveKeeper.BondHooks())
+	if Experimental {
```
Comment (gsk967) [inline app/app.go:479]: Can move `app.IncentiveKeeper` inside experimental

File: app/experimental_appconfig.go [hunk 3]
```diff
@@ -3,93 +3,8 @@
...
-	chunks := strings.Split(EnableSpecificWasmProposals, ",")
-	proposals, err := wasm.ConvertToProposals(chunks)
-	if err != nil {
-		panic(err)
-	}
-
-	return proposals
-}
-
-func (app *UmeeApp) customKeepers(
```
Comment (gsk967) [inline app/experimental_appconfig.go:61]: Can you keep customKeepers as it is, so we can use for leverageKeeper or any future experimental module keepers

**Review by gsk967 — APPROVED** [review]
LGTM, Pre Approving 
Can you check the comments

File: app/experimental_appconfig.go (hunk 3 above)
Comment (robert-zaremba) [reply app/experimental_appconfig.go:61]: I don't think this is needed, we can do if in the app.go, it will be more clear.

File: app/app.go (hunk 2 above)
Comment (robert-zaremba) [reply app/app.go:479]: we can initialize the keeper. It doesn't hurt. There is no code running when you initialize the keeper.
It's important to not initialize the module, and not register the hooks nor the begin/end/init blockers

Comment (gsk967) [reply app/app.go:479]: Okay

**Pull request merged**